template/web/app/paymaster-bundler/ABIs/myNFT.ts

```diff
@@ -0,0 +1,11 @@
+export const myNFTABI = [
```

Comment:
The ABI should be under `_contracts`, please follow the pattern we have for the other example as well.